core/arch/arm/plat-ls/conf.mk

```diff
@@ -38,6 +38,15 @@ $(call force,CFG_CORE_CLUSTER_SHIFT,2)
 CFG_SHMEM_SIZE ?= 0x00200000
 endif
 
+ifeq ($(PLATFORM_FLAVOR),ls1012grapeboard)
+CFG_HW_UNQ_KEY_REQUEST ?= y
+include core/arch/arm/cpu/cortex-armv8-0.mk
+$(call force,CFG_TEE_CORE_NB_CORE,1)
```

Comment:
This file should probably be refactored at some point to put soc-specific configuration in one place.